tcp/tcp.py

```diff
@@ -119,8 +139,10 @@ def __process_requests(self, client_socket, client_addr):
                     logger.info("TCP - is formatted, sending file")
                     break
                 else:
-                    pass
-                    #print("TCP - not supported request")
+                    try:
+                        req_dict[req]() # log changed state if command is valid and not handled above
```

Comment:
i thought i had mentioned this before, but i don't see it in the history. can this exception handling be replace with:

if req in req_dict:
    req_dict[req]()

Reply:
Yes. It has been updated.